doc/integrations/webhooks/Incoming-Webhooks.md

```diff
@@ -0,0 +1,74 @@
+
+_Example:_
+
+Suppose you wanted to create a notification of the status of a daily build, with a table of total tests run and total tests failed by component category, with links to failed tests by category. You could create the following JSON payload to post to a Mattermost channel using webhooks: 
+
+```
+payload={"text": "
+*** 
+##### Build Break - Project X - December 12, 2015 - 15:32 GMT +0
+| Component  | Tests Run   | Tests Failed                                   |
+|:-----------|:------------|:-----------------------------------------------|
+| Server     | 948         | :white_check_mark: 0                           |
+| Web Client | 123         | :warning: [2 (see details)](http://linktologs) |
+| iOS Client | 78          | :warning: [3 (see details)](http://linktologs) |
+***
+"}
+```
+Which would render in a Mattermost message as follows:
+
+*** 
+##### Build Break - Project X - December 12, 2015 - 15:32 GMT +0
+| Component  | Tests Run   | Tests Failed |
+|:------------ |:---------------|:-----|
+| Server | 948       |  :white_check_mark: 0 |
+| Web Client | 123              |   :warning: [2 (see details)](http://linktologs) |
+| iOS Client | 78        |    :warning: [3 (see details)](http://linktologs) |
+*** 
+
+### Creating Integrations using Incoming Webhooks
+You can create a webhook integration to post into Mattermost channels and private groups using these steps: 
+
+1. Enable incoming webhooks from **System Console -> Service Settings**  
+
+ 1. Optionally configure the **Enable Overriding of Usernames from Webhooks** option to allow external applications to post messages under any name. If not enabled, the username of the creator of the webhook URL is used to post messages.
+ 2. Optionally configure the **Enable Overriding of Icon from Webhooks** option to allow external applciations to change the icon of the account posting messages. If not enabled, the icon of the creator of the webhook URL is used to post messages. 
+2. Create a Mattermost Incoming Webhook URL
+ 1. Login to your Mattermost team site and go to **Account Settings -> Integrations**
+ 2. Next to **Incoming Webhooks** click **Edit**
+ 3. Select the channel or private group to receive webhook payloads, then click **Add** to create the webhook
+
+3. Write a function to call the URL with a properly formatted JSON payload
+ 1. To make sure everything works, try a curl command to send a JSON string as the `payload` parameter in a HTTP POST request.
+     1. Example: 
+     ```
+curl -i -X POST -d 'payload={"text": "Hello, this is some text."}' http://yourmattermost.com/hooks/xxx-generatedkey-xxx
+```
+ 2. Incorporate the working webhook into your application to send real time updates to Mattermost channels and private groups.
```

Comment:
This should say something more like "Setup your integration running your on Heroku, an AWS server, or on a machine of your own to start sending real time updates..."